bin/webpack-helpers.js

```diff
@@ -124,6 +125,7 @@ const requestToHandle = ( request ) => {
 module.exports = {
 	NODE_ENV,
 	FORCE_MAP,
+	CIRCULAR_DEPS,
```

Comment:
I appreciate the brevity, but I wonder if we could pick a name that makes it a bit clearer what this does - something like `CIRCULAR_DEPS_CHECK`?